_src/edx-pl/images/edx-svg/svgdefs.svg

```diff
@@ -1,346 +1,366 @@
+</symbol>
+<symbol id="icon-square-o" viewBox="0 0 805 1024">
+	<title>square-o</title>
+	<path class="path1" d="M640 146.286h-475.429q-37.714 0-64.571 26.857t-26.857 64.571v475.429q0 37.714 26.857 64.571t64.571 26.857h475.429q37.714 0 64.571-26.857t26.857-64.571v-475.429q0-37.714-26.857-64.571t-64.571-26.857zM804.571 237.714v475.429q0 68-48.286 116.286t-116.286 48.286h-475.429q-68 0-116.286-48.286t-48.286-116.286v-475.429q0-68 48.286-116.286t116.286-48.286h475.429q68 0 116.286 48.286t48.286 116.286z"></path>
+</symbol>
+<symbol id="icon-square" viewBox="0 0 1024 1024">
+	<title>square</title>
+	<path class="path1" d="M877.714 237.714v548.571q0 68-48.286 116.286t-116.286 48.286h-548.571q-68 0-116.286-48.286t-48.286-116.286v-548.571q0-68 48.286-116.286t116.286-48.286h548.571q68 0 116.286 48.286t48.286 116.286z"></path>
+</symbol>
+<symbol id="icon-circle" viewBox="0 0 1024 1024">
+	<title>circle</title>
+	<path class="path1" d="M877.714 512q0 119.429-58.857 220.286t-159.714 159.714-220.286 58.857-220.286-58.857-159.714-159.714-58.857-220.286 58.857-220.286 159.714-159.714 220.286-58.857 220.286 58.857 159.714 159.714 58.857 220.286z"></path>
+</symbol>
+<symbol id="icon-circle-thin" viewBox="0 0 1024 1024">
+	<title>circle-thin</title>
+	<path class="path1" d="M438.857 146.286q-74.286 0-142 29.143t-116.571 78-78 116.571-29.143 142 29.143 142 78 116.571 116.571 78 142 29.143 142-29.143 116.571-78 78-116.571 29.143-142-29.143-142-78-116.571-116.571-78-142-29.143zM877.714 512q0 119.429-58.857 220.286t-159.714 159.714-220.286 58.857-220.286-58.857-159.714-159.714-58.857-220.286 58.857-220.286 159.714-159.714 220.286-58.857 220.286 58.857 159.714 159.714 58.857 220.286z"></path>
+</symbol>
+<symbol id="icon-search" viewBox="0 0 951 1024">
 	<title>search</title>
 	<path class="path1" d="M658.286 475.429q0-105.714-75.143-180.857t-180.857-75.143-180.857 75.143-75.143 180.857 75.143 180.857 180.857 75.143 180.857-75.143 75.143-180.857zM950.857 950.857q0 29.714-21.714 51.429t-51.429 21.714q-30.857 0-51.429-21.714l-196-195.429q-102.286 70.857-228 70.857-81.714 0-156.286-31.714t-128.571-85.714-85.714-128.571-31.714-156.286 31.714-156.286 85.714-128.571 128.571-85.714 156.286-31.714 156.286 31.714 128.571 85.714 85.714 128.571 31.714 156.286q0 125.714-70.857 228l196 196q21.143 21.143 21.143 51.429z"></path>
 </symbol>
-<symbol id="fa-envelope-o" viewBox="0 0 1024 1024">
+<symbol id="icon-envelope-o" viewBox="0 0 1024 1024">
```

Comment:
Hmm, I thought these had already been synced up to use ``icon-`` vs. ``fa-``. Was I imagining things?